examples/langmuir_turbulence.jl

```diff
@@ -132,7 +132,7 @@ model = NonhydrostaticModel(; grid, coriolis,
                             tracers = :b,
                             buoyancy = BuoyancyTracer(),
                             closure = AnisotropicMinimumDissipation(),
-                            stokes_drift = UniformStokesDrift(∂z_uˢ=∂z_uˢ),
+                            stokes_drift = UniformStokesDrift(∂z_uˢ=∂z_uˢ, parameters = (; vertical_scale)),
```

Comment:
Maybe keep this example simple?

Reply:
I changed this because (iirc) at some point we agreed (on a issue somewhere) that the examples should be able to run on a GPU without any modifications whenever possible. Without this change this example will fail on GPUs, no?

Reply:
We can run and see.

Reply:
I tried running On `#main` and just adding `GPU()` in the grid seems to be enough for the example to run on GPU.

Reply:
Is this what you were implying here? Or did you mean that with the changes in this PR adding the kwarg in the model constructor is necessary?

Reply:
No, you're right. This does run on GPUs since a `vertical_scale` and `Us` and defined and `const`s, which I had missed before.

However, I do think using `parameters` is better since using `const`s prevents a user from playing with this example in the same session by changing values.

I suggested new changes which uses `parameters` to run on GPUs and still simplifies things. lmk what you think.

Reply:
no, `const` is fine for GPU. This is a convenience but not a necessity

Reply:
I'm only hesitant because this design is different than forcing or boundary conditions. In those, `parameters` applies to a single function. Here the design is subtly different because `parameters` applies to every component of the stokes shear. It's not a huge deal but I wonder if it's example-worthy if the design is not final

Reply:
Okay, I'll revert it.